src/pages/Dashboard.tsx

```diff
@@ -53,23 +59,42 @@ export default () => {
     setTransactionResult
   ] = useState<PagedResult<Transaction>>({ rows: [], count: 0 });
 
+  let infoSubscription: Subscription;
+
   useEffect(() => {
-    setBlocksPending(true);
-    fetchBlocks({ startAt: 0, endAt: blocksPerPage }).then(payload => {
-      setBlocksPending(false);
-      setBlockResult(payload);
-    });
-  }, [blocksPerPage]);
+    if (!infoSubscription) {
+      infoSubscription = observeInfo().subscribe(info => {
+        latestInfo.update(info);
+        setSnapshotHeight(info.blockHeight);
```

Comment:
This comes from the web socket connection. Updates Blocks and Transactions every time an info change occurs (blockHeight changes)